docs/support_files/world.md

```diff
@@ -33,4 +33,10 @@ defineSupportCode(function({setWorldConstructor}) {
 });
 ```
 
+If you want to be able to attach screenshots to the results file you will need to pass in {attach} explicity to your CustomWorld.
+
+```javascript
+function CustomWorld({attach}) {...}
+```
```

Comment:
Can we just clarify what the arguments passed into the world constructor are in the example of the default world constructor above?